WooCommerce/Classes/Analytics/WooAnalyticsEvent+FreeTrialSurvey.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+extension WooAnalyticsEvent {
+    enum FreeTrialSurvey {
+
+        private enum Key: String {
+            case source = "source"
+            case surveyOption = "survey_option"
+            case freeText = "free_text"
+        }
+
+        static func surveySent(source: FreeTrialSurveyCoordinator.Source,
+                               surveyOption: String?,
+                               freeText: String?) -> WooAnalyticsEvent {
```

Comment:
❓ Why don't we simplify the logic by keeping survey option non-optional? For the "other" option, we can require free text to submit the answer.

Also a suggestion: Please align the slug with Android for the survey option to ensure the values match for both platforms. Last time I looked into the onboarding survey results, I struggled because each platform sent different values for the same options. Aligning the values will help us later on when analyzing data.

Reply:
> Why don't we simplify the logic by keeping survey option non-optional? 

Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/598930902d59b5dd0dbb2c2c692b0b2ca2da8232

> For the "other" option, we can require free text to submit the answer.

I will do this when I implement submit in a future PR. 

> Also a suggestion: Please align the slug with Android for the survey option to ensure the values match for both platforms. Last time I looked into the onboarding survey results, I struggled because each platform sent different values for the same options. Aligning the values will help us later on when analyzing data.

I understand. Thanks for pointing this out. These values are from the PT, and we discussed it in this Slack thread. p1690269742291779-slack-C03L1NF1EA3 (i.e. We are in sync with Android.)